package.json

```diff
@@ -57,28 +48,24 @@
   },
   "homepage": "https://github.com/taion/graphql-type-json#readme",
   "devDependencies": {
-    "@babel/cli": "^7.12.13",
-    "@babel/core": "^7.12.13",
-    "@babel/preset-env": "^7.12.13",
-    "babel-jest": "^26.6.3",
-    "codecov": "^3.8.1",
-    "cpy-cli": "^3.1.1",
-    "dtslint": "^3.7.0",
-    "eslint": "^7.19.0",
-    "eslint-config-4catalyzer": "^1.1.5",
-    "eslint-config-4catalyzer-jest": "^2.0.10",
-    "eslint-config-prettier": "^6.15.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^23.20.0",
-    "eslint-plugin-prettier": "^3.3.1",
+    "codecov": "^3.8.3",
+    "eslint": "^8.10.0",
+    "eslint-config-4catalyzer": "^1.4.1",
+    "eslint-config-4catalyzer-jest": "^2.2.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-jest": "^26.1.1",
+    "eslint-plugin-prettier": "^4.0.0",
     "graphql": "^15.5.0",
-    "husky": "^4.3.8",
-    "jest": "^26.6.3",
-    "lint-staged": "^10.5.4",
-    "prettier": "^2.2.1",
-    "typescript": "^4.1.3"
+    "husky": "^7.0.4",
+    "jest": "^27.5.1",
+    "lint-staged": "^12.3.5",
+    "prettier": "^2.5.1",
+    "rimraf": "^3.0.2",
+    "ts-jest": "^27.1.3",
+    "typescript": "^4.6.2"
   },
   "peerDependencies": {
-    "graphql": ">=0.8.0"
+    "graphql": ">=0.8.0 < 16"
```

Comment:
https://github.com/graphql/graphql-js/blob/v15.8.0/docs/APIReference-TypeSystem.md

Support up to 15.x